homeassistant/components/philips_js/switch.py

```diff
@@ -70,3 +76,57 @@ async def async_turn_on(self, **kwargs: Any) -> None:
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn the entity off."""
         await self.coordinator.api.setScreenState("Off")
+
+
+class PhilipsTVAmbilightHueSwitch(
+    CoordinatorEntity[PhilipsTVDataUpdateCoordinator], SwitchEntity
+):
+    """A Philips TV Ambi+Hue switch."""
+
+    def __init__(
+        self,
+        coordinator: PhilipsTVDataUpdateCoordinator,
+    ) -> None:
+        """Initialize entity."""
+
+        super().__init__(coordinator)
+
+        self._attr_name = f"{coordinator.system['name']} Ambilight+Hue"
+        self._attr_icon = "mdi:television-ambient-light"
+        self._attr_unique_id = f"{coordinator.unique_id}_ambi_hue"
+        self._attr_is_on = self.coordinator.api.huelamp_power == HUE_POWER_ON
```

Comment:
This can be removed. The property is_on handled this. You can also remove the handle coordinator update function